.github/workflows/redis-stack-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Redis stack tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+
+  predis:
+    name: PHP ${{ matrix.php-versions }} (Redis stack ${{ matrix.redis-stack-versions }})
+    runs-on: ubuntu-latest
+
+    services:
+      redis:
+        image: redis/redis-stack-server:${{ matrix.redis-stack-versions }}
+        ports:
+        - 6379:6379
+        options: --health-cmd="redis-cli ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4', '8.0', '8.1', '8.2']
+        redis-stack-versions: ['6.2.6-v0', '7.0.6-RC2']
```

Comment:
IMHO you shouldn't have to do this. 'latest' should be fine - especially since the redis version isn't he important change, but the underlying module versions.

redis-stack-server releases monthly, so I'd err on the side of latest here - like other clients.

Reply:
Okay, got it. I just followed by approach of current workflow that runs tests over each major version of Redis server